plugins/protected-branch/src/index.ts

```diff
@@ -100,5 +100,22 @@ export default class ProtectedBranchPlugin implements IPlugin {
         );
       }
     );
+
+    auto.hooks.afterRelease.tapPromise(this.name, async () => {
+      if (!auto.git) {
+        return;
+      }
+
+      const sha = await auto.git.getSha();
+      const headBranch = `${this.options.releaseTemporaryBranchPrefix}${sha}`;
+
+      auto.logger.log.info("Delete release branch 🗑️ ");
+
+      await auto.git.github.git.deleteRef({
```

Comment:
👋
don't think it is needed. The PR is merged via github, then the github policy should be applied when a PR is merged 🤷